collector/perf-runner.md

```diff
@@ -42,11 +42,12 @@ The machine that actually executes the benchmarks in CI has the following config
 - OS: Ubuntu `18.04`
 
 ## Configuration
-### Turboboost
-Enabled.
+### HyperThreading & Turboboost
+As of November 2022, both are disabled.
 
-### HyperThreading
-Enabled.
+Prior to October 2022, both were enabled. During October 2022 some
```

Comment:
We could add a link to https://github.com/rust-lang/rustc-perf/issues/1450 here.